walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,202 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.HashMap;
 
 public final class FindMeetingQuery {
+
+  /**
+   * This method checks if an event contains no mandatory attendees using a MeetingRequest
+   * @param request MeetingRequest object which contains list of attendees
+   * @param event The Event object's attendees are compare with the request object
+   * @return boolean if the provide event object contains no mandatory attendees
+   */
+  private boolean eventNoMandatoryAttendees(MeetingRequest request, Event event){
```

Comment:
nit: For methods that return a boolean, I've found it helpful to make that clear in the name of the function. Something like `hasNoMandatoryAttendees` could improve readability.

Reply:
Ok